api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -252,4 +256,26 @@ public Concept getOrderGroupReason() {
 	public void setOrderGroupReason(Concept orderGroupReason) {
 		this.orderGroupReason = orderGroupReason;
 	}
+	
+	/**
+	 * Gets the previous order group to other order groups, to maintain linkages
+	 * between groups and support group nesting
+	 * 
```

Comment:
Kindly also take a look at some spaces being left otherwise thanks for your work

Reply:
i think the spaces are okay as you can see from the rest of the docs in the class ..Thanks

Reply:
@gitcliff thanks